src/app/organizations/vault/vault.component.ts

```diff
@@ -263,6 +263,18 @@ export class VaultComponent implements OnInit, OnDestroy {
         return childComponent;
     }
 
+    cloneCipher(cipher: CipherView) {
+        const component = this.editCipher(cipher);
+        component.cloneMode = true;
+        component.organizationId = this.organization.id;
+        if (this.organization.isAdmin) {
+            component.collections = this.groupingsComponent.collections.filter((c) => !c.readOnly);
+        }
+        // Regardless of Admin state, the collection Ids need to passed manually as they are not assigned value
+        // in the add-edit componenet
+        component.collectionIds = cipher.collectionIds;
```

Comment:
What if the cloning user doesn't have access to assign the items to the existing `cipher.collectionIds`? I say we just use the same logic as the addCipher:

```
if (this.collectionId != null) {
	component.collectionIds = [this.collectionId];
}
```

Maybe I am missing something though.

Reply:
In the add flow as a nicety, we assume that if they are viewing a specific collection and hit "add item" button, they want to add it to the currently view collection. 

However, it is different during the clone flow because technically we don't care about what's being viewed, rather, we care about the collections specific to the cipher being cloned.

To address your question: because we made the change to allow clone mode to manually filter the passed collections, if a user does not have access to assign items to any collections, they will be filtered out from the list.